it/test/uk/gov/hmrc/tradergoodsprofilesdatastore/connectors/CustomsDataStoreConnectorSpec.scala

```diff
@@ -91,4 +91,41 @@ class CustomsDataStoreConnectorSpec
       connector.getEmail(testEori).failed.futureValue
     }
   }
+
+  ".getEoriHistory" - {
+
+    "must return eori history" in {
+
+      val mockEoriHistoryResponse = EoriHistoryResponse(
+        Seq(
+          EoriHistoricItem("eori1", Instant.parse("2024-02-20T00:00:00Z"), Instant.parse("2024-01-20T00:00:00Z")),
+          EoriHistoricItem("eori1", Instant.parse("2024-03-20T00:00:00Z"), Instant.parse("2024-01-20T00:00:00Z")),
+          EoriHistoricItem("eori1", Instant.parse("2024-01-20T00:00:00Z"), Instant.parse("2024-01-20T00:00:00Z"))
+        )
+      )
+      wireMockServer.stubFor(
+        get(urlEqualTo(s"/customs-data-store/eori/$testEori/eori-history"))
+          .willReturn(ok().withBody(Json.toJson(mockEoriHistoryResponse).toString()))
+      )
+
+      val expectedResponse = EoriHistoryResponse(
+        Seq(
+          EoriHistoricItem("eori1", Instant.parse("2024-03-20T00:00:00Z"), Instant.parse("2024-01-20T00:00:00Z")),
+          EoriHistoricItem("eori1", Instant.parse("2024-02-20T00:00:00Z"), Instant.parse("2024-01-20T00:00:00Z")),
+          EoriHistoricItem("eori1", Instant.parse("2024-01-20T00:00:00Z"), Instant.parse("2024-01-20T00:00:00Z"))
+        )
+      )
+
+      connector.getEoriHistory(testEori).futureValue mustEqual Some(expectedResponse)
+    }
+
```

Comment:
there is no tests for 'none' which probably means this doesn't need to be an option